gen/generate.jl

```diff
@@ -22,7 +22,7 @@ wc = wrap_c.init(
     clang_extraargs,
     (th, h) ->
     begin
-        if search(h, "expat") == 0:-1
+        if something(findfirst("expat", h), 0:-1) == 0:-1
```

Comment:
This works, but it would be much more logical to write this instead:
```julia
if findfirst("expat", h) === nothing
```
You could even replace all this block with `occursin("expat", h)`.

Reply:
Yeah I just replaced it with the exact suggestion from the deprecation warning that Julia 0.7 gave me, just to make sure I didn't break anything.